components/brave_wallet_ui/components/desktop/hardware-wallet-connect/accounts_list.tsx

```diff
@@ -261,63 +220,43 @@ export const HardwareWalletAccountsList = ({
                 </HelpLink>
               </Row>
               <div slot='value'>
-                {getDerivationPathLabel(
-                  selectedDerivationScheme as
-                    | LedgerDerivationPath
-                    | TrezorDerivationPath
-                )}
+                {getHardwareImportSchemeLabel(currentHardwareImportScheme)}
               </div>
-              {Object.keys(ethDerivationPathsEnum).map((path) => {
-                const pathValue = getPathValue(path)
-
+              {supportedSchemes.map((scheme) => {
                 return (
                   <leo-option
-                    value={pathValue}
-                    key={pathValue}
+                    value={scheme.derivationScheme}
+                    key={scheme.derivationScheme}
                   >
-                    {getDerivationPathLabel(pathValue)}
+                    {getHardwareImportSchemeLabel(scheme)}
                   </leo-option>
                 )
               })}
             </Dropdown>
           ) : null}
           {coin === BraveWallet.CoinType.SOL ? (
             <Dropdown
-              value={selectedDerivationScheme}
+              value={currentHardwareImportScheme.derivationScheme}
               onChange={(e) => onChangeDerivationScheme(e.value)}
             >
               <div slot='value'>
-                {
-                  solDerivationPathsEnum[
-                    selectedDerivationScheme as SolDerivationPaths
-                  ]
-                }{' '}
-                {`"${getPathForSolLedgerIndex(
-                  undefined,
-                  selectedDerivationScheme as SolDerivationPaths
-                )}"`}
+                {getHardwareImportSchemeLabel(currentHardwareImportScheme)}
               </div>
-              {Object.keys(solDerivationPathsEnum).map((path) => {
-                const pathLocale =
-                  solDerivationPathsEnum[path as SolDerivationPaths]
+              {supportedSchemes.map((scheme) => {
                 return (
                   <leo-option
-                    value={path}
-                    key={path}
+                    value={scheme.derivationScheme}
+                    key={scheme.derivationScheme}
                   >
-                    {pathLocale}{' '}
-                    {`"${getPathForSolLedgerIndex(
-                      undefined,
-                      path as SolDerivationPaths
-                    )}"`}
+                    {getHardwareImportSchemeLabel(scheme)}
                   </leo-option>
                 )
               })}
             </Dropdown>
           ) : null}
         </SelectWrapper>
       </SelectRow>
-      {coin !== BraveWallet.CoinType.FIL && (
+      {[BraveWallet.CoinType.ETH, BraveWallet.CoinType.SOL].includes(coin) && (
```

Comment:
perf: can we define this array outside of the component to avoid recreating it on each render?